03-a_review_of_r_modeling_fundamentals.Rmd

```diff
@@ -235,10 +256,10 @@ reg_fits <-
   trees2 %>%
   group_nest(group) %>% 
   mutate(
-    fit = map(data, ~ lm(formula(Volume ~ .), data = .x)),
-    tidied = map(fit, tidy),
-    glanced = map(fit, glance),
-    augmented = map(fit, augment)
+    fit = map(data, ~ lm(Volume ~ ., data = .x)),
+    tidied = map(fit, tidy), #converts model object's coefficients to a dataframe
+    glanced = map(fit, glance), #row summary of a model
+    augmented = map(fit, augment) #returns a tibble of additional metrics like Cooks distance, lower and upper bounds of fitted values, standard errors of fitted values
```

Comment:
My personal preference would be for the comments to be above each line, and keep them wrapped at 80 characters for readability (ctrl-shift-/ in Rstudio).